tests/integ/test_udf.py

```diff
@@ -84,6 +84,20 @@ def setup(session, resources_path):
     )
 
 
+def return1():
+    return "1"
+
+
+@pytest.mark.skipif(
+    IS_IN_STORED_PROC, reason="Source code extraction fails in stored proc"
```

Comment:
Will the error message change for stored proc and what is the new error message looks like?

Reply:
The error message is something like "`<handler>_module` is not found" and happens when user calls the stored procedure.

Reply:
As discussed, let's try to catch and throw a more actionable error message for stored proc user, for example, suggest them to move the function/variable into the handler function. I personally don't think it should be a behavior change, because we just make an unclear random error message better and more actionable 🤔

Reply:
so it's actually not working in stored proc after we change to use `inspect.getsource`? Then the original issue in the jira is not solved? Could you run this branch with Stored procedure regression test?

Reply:
It never worked in stored proc, `inspect.getsource` depends on `linecache` which assumes the code is loaded from a file, this is not true in `UDF`. The original issue is not solved :( the fix enables non-local function definitions outside stored procedures.

Here's the precommit: https://ci-dev-133.int.snowflakecomputing.com/job/BuildAndUpload-Precommit/4067/

Reply:
Great suggestion!
Switched this to [SNOW-707132](https://snowflakecomputing.atlassian.net/browse/SNOW-707132).
Created [SNOW-707162](https://snowflakecomputing.atlassian.net/browse/SNOW-707162), which then should fix the original issue.